src/transformers/configuration_barthez.py

```diff
@@ -0,0 +1,103 @@
+# coding=utf-8
+# Copyright 2020 The Fairseq Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" BARThez configuration """
+
+from .configuration_bart import BartConfig
+from .utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+BARTHEZ_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "moussaKam/barthez": "https://s3.amazonaws.com/models.huggingface.co/bert/moussaKam/barthez/config.json",
```

Comment:
```suggestion
    "moussaKam/barthez": "https://huggingface.co/moussaKam/barthez/resolve/main/config.json",
```

Reply:
Update URL to current scheme (even though those urls shouldn't be used anymore...)

Reply:
Done in 0a77338